libtiledbsoma/include/tiledbsoma/soma_array_reader.h

```diff
@@ -374,4 +374,4 @@ class SOMAArrayReader {
 
 }  // namespace tiledbsoma
 
-#endif
+#endif  // SOMA_ARRAY_READER
```

Comment:
idk how people feel about this comment after `#endif` but it's a long-held personal preference of mine ...

https://github.com/johnkerl/scripts/blame/main/fundam/hdrtop

Reply:
It's been a long day so I am even more dim-witted than usual.  At the link I see

![image](https://user-images.githubusercontent.com/673121/228983696-da8f07c5-c579-46a8-8769-f4967c49112d.png)

and now I do not know you are talking about.  "Talk to me like I am five."

Reply:
@eddelbuettel sorry to be cryptic.

Just an 11-year-old script which shows that every time I create a header file I want to do `:.!hdrtop %` which makes

```
#ifndef FOO_H
#define FOO_H

#endif // FOO_H
```

showing that I've had this stylistic preference for long enough to have automated around it. :)

Reply:
Superflouous at the end of a file like this that has one and only exactly one header guard.

But in principle I like comments at the end of long and nested 'end' or closing brackets.

Lastly for the case of header guards:  the cool kids switched to `#pragma once` to not have this problem

Very lastly too minor an issue to worry.  But if you feel strongly about it and have the urge to touch umpteen files go for it.